pkg/apis/troubleshoot/v1beta2/collector_shared.go

```diff
@@ -123,6 +123,16 @@ type Database struct {
 	URI           string `json:"uri" yaml:"uri"`
 }
 
+type RRD struct {
+	CollectorMeta   `json:",inline" yaml:",inline"`
+	Namespace       string            `json:"namespace" yaml:"namespace"`
+	Image           string            `json:"image" yaml:"image"`
```

Comment:
is this needed  or why would someone specify this ?

Reply:
I don't want to hardcode any images in troubleshoot.  But it _will_ be optional. We'll inject either a kotsadm image here or use some public image in default static collector.  But in case someone wants to use this from CLI with their own spec, they could specify an image here.